ariadne/contrib/HuggingFaceTransformer.py

```diff
@@ -0,0 +1,29 @@
+"""
+@author: Ghadeer Mobasher
+"""
+from cassis import Cas
+from ariadne.contrib.inception_util import create_prediction, SENTENCE_TYPE
+from transformers import pipeline
+from transformers import AutoTokenizer, AutoModelForTokenClassification
+import pandas as pd
+from ariadne.classifier import Classifier
+
+class HuggingFaceClassifier(Classifier):
+    '''As an example, to use it HuggingFace models for token classification  
+    HuggingFaceClassifier(model_name="ghadeermobasher/BC5CDR-Chemical-Disease-balanced-pubmedbert")'''
+    def __init__(self, model_name: str):
+            super().__init__()
+            self._model = model_name
+    def predict(self, cas: Cas, layer: str, feature: str, project_id: str, document_id: str, user_id: str):
+        tokenizer = AutoTokenizer.from_pretrained(self._model)
+        model = AutoModelForTokenClassification.from_pretrained(self._model)
+        nlp_ner = pipeline("ner", model=model, tokenizer=tokenizer,aggregation_strategy="max")
+        for c, sentence in enumerate(cas.select(SENTENCE_TYPE)):
```

Comment:
I can't see the `c` being used. If it is not needed, I guess the `enumerate` is not needed either?

Reply:
Yes, true. I was using them for other purposes and forgot to remove them.